### ISSUE TITLE
[Linux] Only first battery reported with psutil.sensors_battery()

### ISSUE THREAD

**Issue by VorpalBlade** [giampaolo/psutil#1460]
**Linux**
* Ubuntu 16.04 x86_64
* psutil version 5.6.1

**Bug description**

Only linux the first batter is reported. Some laptops (such as Lenovo Thinkpad T480) has two batteries, one built in, and one removable. The information about the removable battery is not reported
Example:
```
In [7]: psutil.sensors_battery()                                                                                                                                                                                                     
Out[7]: sbattery(percent=92.84843559528647, secsleft=<BatteryTime.POWER_TIME_UNLIMITED: -2>, power_plugged=True)
```

But two batteries (and AC) are available in the sys filesystem:
```
$ ls /sys/class/power_supply/
AC  BAT0  BAT1
```

**Comment by giampaolo** [comment]
Yeah, that's by design (at the moment). We may want to introduce a `all=False` parameter or something. When True it should return a list of namedtuples.

**Comment by amanusk** [comment]
@giampaolo I can try and take a look at this

**Comment by giampaolo** [comment]
Go for it Alex.

**Comment by mohkale** [comment]
I'm using psutil to create components for my tmux status-line and it seems to be picking up the baterry level for my bluetooth keyboard by default :joy:, any idea if #1466 might be merged soon?
